American-Elections-and-Crypto.md

```diff
@@ -0,0 +1,107 @@
+
+“I'm a big fan of our currency and I don't want to have other currencies coming out and hurting or demeaning the dollar in any way. If you look at a monetary system based on the dollar, if you start losing credibility, all of a sudden you're going to lose that strong monetary system, we have to be very careful about that.”
+
+Trump also criticized Bitcoin during a Fox News interview in the same year. Showing once more a vision of protectionism when regarding the American dollar:
+
+"The currency of this world should be the dollar. And I don't think we should have all of the Bitcoins of the world out there. I think they should regulate them very, very high."
+
+It's challenging to predict how Trump would act in relation to crypto. While his involvement in launching digital assets like NFTs and holding crypto may suggest a favorable stance, it's impossible to make a definitive prediction.
+
+## Will anything really change?
+
+Both the Biden and Trump administrations, while having divergent views on cryptocurrency, share a common objective: to ensure that traditional financial institutions retain control and influence over the financial landscape, even as digital currencies gain traction. Despite potential differences in regulatory approaches, both administrations are likely to prioritize measures that maintain the supremacy of the American dollar and protect the interests of established financial entities.
+
+Traditional American financial institutions wield significant power and influence within the American and global economy. They are unlikely to cede control to cryptocurrencies without some form of regulation or oversight that aligns with their interests. Both administrations are aware of the potential disruptive impact of cryptocurrencies on traditional financial systems and are thus inclined to implement measures that mitigate this risk. 
+
+Big firms like BlackRock and Fidelity are already gaining some control over  Bitcoin, holding approximately 0,5% of the total supply, but the majority of their investments will always remain in the US. Dollar. In response to the potential tightening of regulations under either administration, cypherpunks may prioritize the development and adoption of privacy-enhancing technologies, decentralized finance (DeFi) platforms, and censorship-resistant communication tools. By focusing on technologies that empower individuals to transact and communicate freely, cypherpunks can continue to advance their vision of a more open and equitable digital future, regardless of the political climate.
+
+Moreover, cypherpunks may engage in advocacy efforts to educate policymakers and the public about the benefits of decentralized systems and the risks of excessive regulation. By participating in grassroots campaigns, lobbying efforts, and community outreach, cypherpunks can work to shape the regulatory environment in a way that fosters innovation and protects individual freedoms.
```

Comment:
I think this is much better now, after your edits. It's a shame that the basic outcome of this event is the status quo but to be honest that was my initial thought. Once you've completed the final edit, upload to CT. Let me know when it's ready.